crates/db_views/src/structs.rs

```diff
@@ -82,29 +93,60 @@ use ts_rs::TS;
 
 #[skip_serializing_none]
 #[derive(Debug, PartialEq, Serialize, Deserialize, Clone)]
-#[cfg_attr(feature = "full", derive(TS, Queryable))]
+#[cfg_attr(feature = "full", derive(TS, Queryable, Selectable))]
 #[cfg_attr(feature = "full", diesel(check_for_backend(diesel::pg::Pg)))]
 #[cfg_attr(feature = "full", ts(export))]
 /// A comment report view.
 pub struct CommentReportView {
+  #[cfg_attr(feature = "full", diesel(embed))]
   pub comment_report: CommentReport,
+  #[cfg_attr(feature = "full", diesel(embed))]
   pub comment: Comment,
```

Comment:
This file contains all the selects for the views now. Its as simple as adding `diesel(embed)` for most of the tables (except the aliased ones, and custom selects)`